pkgs/development/libraries/fmt/0001-add-fmt.pc.patch

```diff
@@ -0,0 +1,30 @@
+From 7ad56220fd23cfce7c1c3e3bc269fa88d34070eb Mon Sep 17 00:00:00 2001
```

Comment:
This looks like you have send this also upstream? Can you include a link to the pull request if that's the case?

Reply:
(I've now added it next to the patch line in `default.nix`, if that's okay.)